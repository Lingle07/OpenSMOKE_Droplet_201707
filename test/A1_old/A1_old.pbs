#!/bin/bash

#PBS -N A1_old
#PBS -e A1_old.err 
#PBS -o A1_old.out 
#PBS -l nodes=2:ppn=24 
#PBS -V 

PBS_O_WORKDIR='/home/xyhuang/OpenSMOKE_Droplet/exe/' 
export PBS_O_WORKDIR 
### --------------------------------------- 
### BEGINNING OF EXECUTION 
### --------------------------------------- 

echo The master node of this job is \`hostname\` 
echo The working directory is \`echo $PBS_O_WORKDIR\` 
echo This job runs on the following nodes: 
echo \`cat $PBS_NODEFILE\` 

### end of information preamble 

cd $PBS_O_WORKDIR

cmd="./OpenSMOKEpp_MicrogravityDroplet.sh --input ./test_simulation/input.A1.dic"

echo "Running Intel-15 MVAPICH linked bounce with: $cmd in directory "`pwd`

$cmd >& /home/xyhuang/OpenSMOKE_Droplet/exe/jobs/log.$PBS_JOBID
