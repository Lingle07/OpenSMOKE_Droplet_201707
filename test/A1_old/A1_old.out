The master node of this job is `hostname`
The working directory is `echo /home/xyhuang/OpenSMOKE_Droplet/exe/`
This job runs on the following nodes:
`cat /var/spool/torque/aux//202159.certainty-fe.stanford.edu`
Running Intel-15 MVAPICH linked bounce with: ./OpenSMOKEpp_MicrogravityDroplet.sh --input ./test_simulation/input.A1.dic in directory /home/xyhuang/OpenSMOKE_Droplet/exe
